RELEASE.rst

```diff
@@ -0,0 +1,15 @@
+RELEASE_TYPE: minor
+
+This release adds utils for the reporting pipeline. 
+
+The functions under ``reporting_utils.py`` describe a basic ETL pipeline from VHS to elasticsearch, without a transformation specified. In this way, the shape of the pipeline remains independent of both the data within it and the transforms being applied. 
+
+As further data sources are added to the reporting pipeline and more lambdas are created, we keep repeated code to a minimum. In a new lambda, the user should specify a set of data-source-specific transformations in a ``transform.py`` file. The lambda's ``main`` can then remain minimal and generic:
```

Comment:
I would capitalise Lambda just to be clear you’re not talking about anonymous functions in Python.

```suggestion
As further data sources are added to the reporting pipeline and more Lambda functions are created, we keep repeated code to a minimum. In a new Lambda function, the user should specify a set of data-source-specific transformations in a ``transform.py`` file. The Lambda's ``main`` can then remain minimal and generic:
```